test/e2e/integration-services/device-account.spec.bak

```diff
@@ -0,0 +1,320 @@
+import { kratosAdmin } from "@services/kratos/private"
```

Comment:
Why `.bak`?

Reply:
removed it. was keeping it temporarily as a reference to older code to look back at.